komfydb/execution/aggregator.h

```diff
@@ -53,6 +53,16 @@ class Aggregator {
         return "count";
     }
   }
+
+  static bool IsApplicableToString(AggregateType type) {
```

Comment:
I think `IsApplicable(AggregateType aggregate_type, Type type)` would be more extendible.